helios-services/src/test/java/com/spotify/helios/agent/TaskRunnerTest.java

```diff
@@ -58,6 +69,11 @@
   @Mock private Clock clock;
   @Mock private ContainerDecorator containerDecorator;
 
+  @Mock private TaskRunner.NopListener mockListener;
```

Comment:
+1 - I tried to do something similar using the @Mock annotation, but was told off by IntelliJ.